packages/dtslint/test/no-bad-reference.test.ts

```diff
@@ -0,0 +1,62 @@
+import { ESLintUtils } from "@typescript-eslint/utils";
+
+import * as noBadReference from "../src/rules/no-bad-reference";
+
+const ruleTester = new ESLintUtils.RuleTester({
+  parser: "@typescript-eslint/parser",
+});
+
+ruleTester.run("no-bad-reference", noBadReference, {
+  invalid: [
+    {
+      code: `/// <reference path="../other" />`,
+      errors: [
+        {
+          column: 20,
+          endColumn: 28,
+          line: 1,
+          messageId: "referencePathTest",
+        },
+      ],
+      filename: "types.ts",
+    },
+    {
+      code: `/// <reference path="other" />`,
```

Comment:
also should test `./other` to make sure that it continues to work, because that's the most common usage